Desktop/analysis/terms.cpp

```diff
@@ -391,6 +391,42 @@ Terms Terms::ffCombinations(const Terms &terms)
 	return newTerms;
 }
 
+Terms Terms::combineTerms(JASPControl::CombinationType type)
+{
+	Terms combinedTerms;
+
+	size_t nbTerms = size();
+
+	switch (type)
+	{
+	case JASPControl::CombinationType::CombinationCross:
+		combinedTerms = crossCombinations();
+		break;
```

Comment:
newlines after each `break` would have been nice but I won't block the PR on that :p